common/src/main/java/com/redhat/cloud/notifications/models/EventType.java

```diff
@@ -34,6 +35,17 @@
 @JsonFilter(ApiResponseFilter.NAME)
 public class EventType {
 
+    public static final Map<String, String> SORT_FIELDS = Map.of(
+            "name", "name",
```

Comment:
`name` might need to be changed to `e.name` (I'm not sure):
```suggestion
            "name", "e.name",
```